.github/workflows/upgrade-sdk.yml

```diff
@@ -0,0 +1,44 @@
+name: Update @cord-sdk/react package
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 * * * *' # Run every hour
+
+jobs:
+  update-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          # checkout pull request HEAD commit instead of merge commit
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Does this work?  This doesn't run in response to a pull request, so does the event have a `pull_request` property?